src/components/text-input/index.md.njk

```diff
@@ -67,6 +67,8 @@ If you're asking the user to enter a whole number and you want to bring up the n
 
 {{ example({group: "components", item: "text-input", example: "number-input", html: true, nunjucks: true, open: false, size: "m"}) }}
 
+You should also [turn off HTML5 validation](/patterns/validation/#turn-off-html5-validation) to prevent browsers from validating the `pattern` attribute.
```

Comment:
It's unfortunate that the word `pattern` here could lead to some confusion with the 'specific patterns' guidance below. This seems like the most logical place for the HTML5 validation note so maybe the 'specific patterns' section should have its own subheading.

Reply:
Good spot, agree this could be confusing. I wonder if we could change it to something like:

> There is specific guidance on how to ask for:
> 
> - [dates](/patterns/dates/)
> - [telephone numbers](/patterns/telephone-numbers/)

Reply:
@36degrees That's a good idea 👍